specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -908,11 +904,26 @@
     },
     "IndexBatch": {
       "properties": {
-        "name": {
-          "type": "string"
+        "value": {
+          "x-ms-client-name": "Actions",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IndexAction"
+          }
         }
-      },
-      "x-ms-external": true
+      }
+    },
+    "IndexAction": {
+      "properties": {
+        "actionType": {
+          "$ref": "#/definitions/IndexActionType",
+          "x-nullable": false
+        },
+        "document": {
```

Comment:
There is no document property here. This needs to be modeled with `additionalProperties`.